frontend/app/model/Benefits.scala

```diff
@@ -58,57 +55,64 @@ object Benefits {
 
   val friendBenefitsList = benefitsFilter(
     "book_tickets",
-    "digital_digest",
-    "video_highlights"
+    "offers_competitions",
```

Comment:
@tudorraul I'd definitely prefer not to use strings, how would you approach this?